src/applications/hca/components/HCAEnrollmentStatusFAQ.jsx

```diff
@@ -42,11 +42,10 @@ const HCAEnrollmentStatusFAQ = ({
   showingReapplyForHealthCareContent,
   showReapplyContent,
 }) => {
-  const applyAllowed =
-    enrollmentStatus === HCA_ENROLLMENT_STATUSES.activeDutyHasNotApplied;
+  const applyAllowed = enrollmentStatus === HCA_ENROLLMENT_STATUSES.activeDuty;
```

Comment:
A lot of this PR is simply:
- changing `activeDutyHasNotApplied` to `activeDuty`
- removing all handling of `activeDutyHasApplied`